apps/cms.c

```diff
@@ -219,6 +217,9 @@ const OPTIONS cms_options[] = {
 
     OPT_R_OPTIONS,
     OPT_V_OPTIONS,
+
+    OPT_PARAMETERS(),
+    {"cert", 0, 0, "Recipient certs, used when encrypting"},
```

Comment:
perhaps `Recipient certs (optional, required and used only when encrypting)` ?

Reply:
I did "Recipient certs (optional; required only when encrypting)"

Reply:
That wording isn't great either, since you can use -recip.  Sigh. So I made it "used only"